src/ng-generate/components/table/generators/components/table/files/__name@dasherize__.component.html.template

```diff
@@ -96,10 +96,13 @@
                 </th>
 
                 <td data-test="table-cell"
+                    mat-cell *matCellDef="let row"
+                    [matTooltipDisabled]="<%= isEmptyValue %>"
                     <% if(options.templateHelper.isEnumPropertyWithEntityValues(value.property)) { %>
-                        [matTooltip]="!('<%= isEmptyValue %>') ? (row.<%= cellPropertyPath %><%= descriptionPipe %>:true<%= language %>) : ''" [matTooltipDisabled]="'<%= isEmptyValue %>'"
+                        [matTooltip]="!(<%= isEmptyValue %>) ? (row.<%= cellPropertyPath %><%= descriptionPipe %>:true<%= language %>) : ''"
```

Comment:
This should be better since it should generate less code:
```suggestion
                        <%= !isEmptyValue && "[matTooltip]=\"row." + cellPropertyPath + descriptionPipe + ":true" +  language + "\"" %>
```

Reply:
Thanks! Updated